src/main/java/org/elasticsearch/common/jna/Kernel32Library.java

```diff
@@ -37,17 +36,21 @@
 
     private static ESLogger logger = Loggers.getLogger(Kernel32Library.class);
 
-    private Kernel32 internal;
+    private boolean loaded = false;
```

Comment:
and i guess where we instantiate singleton, we just let Kernel32Library be null if it cant load? I dont know if its cleaner actually, it means callers woudl have to check for that. i just think its wierd to have all methods have to check isLoaded and do nothing. Currently there are only a few, but if the class increases...